docs/eventing/samples/writing-event-source-easy-way/README.md

```diff
@@ -0,0 +1,305 @@
+---
+title: "Writing an Event Source using the easy way"
+linkTitle: "Event Source - easy way"
+weight: 10
+type: "docs"
+---
+
+# Introduction
+
+As stated in [tutorial on writing a Source with a Receive Adapter](../writing-receive-adapter-source/README.md), there are multiple ways to
+create event sources. The way in that tutorial is to create an independent event source that has its own CRD.
+
+In this tutorial though, you will build an event source in Javascript and use it with
+[ContainerSource](../../../eventing/sources/README.md#meta-sources) and / or [SinkBinding](../../../eventing/sources/README.md#meta-sources).
+
+[ContainerSource](../../../eventing/sources/README.md#meta-sources) is an easy way to turn any dispatcher container into an Event Source.
+Similarly, another option is using [SinkBinding](../../../eventing/sources/README.md#meta-sources)
+which provides a framework for injecting environment variables into any Kubernetes resource which has a `spec.template` that looks like a Pod (aka PodSpecable).
+
+Code for this tutorial is available at [Github](https://github.com/aliok/node-knative-heartbeat-source).
```

Comment:
Does anybody have an opinion on where to put this repository?

Reply:
what about putting it under `https://github.com/knative/docs/tree/master/docs/eventing/samples` ?

Reply:
hmm, ok, sounds good. I haven't seen any code in `docs/eventing/samples` but there's some in `docs/serving/samples`. Let me add there.